aspnetcore/security/samesite.md

```diff
@@ -0,0 +1,167 @@
+| <xref:Microsoft.AspNetCore.Http.CookieBuilder> | <xref:Microsoft.AspNetCore.Http.CookieBuilder.Name> | `None` |
+| <xref:System.Web.HttpContext.Session>  | [SessionOptions.Cookie](xref:Microsoft.AspNetCore.Builder.SessionOptions.Cookie) |`Lax` |
+| <xref:Microsoft.AspNetCore.Mvc.ViewFeatures.CookieTempDataProvider>  | [CookieTempDataProviderOptions.Cookie](xref:Microsoft.AspNetCore.Mvc.CookieTempDataProviderOptions.Cookie) | `Lax` |
+| <xref:Microsoft.AspNetCore.Antiforgery.IAntiforgery> | [AntiforgeryOptions.Cookie](xref:Microsoft.AspNetCore.Antiforgery.AntiforgeryOptions.Cookie)| `Strict` |
+| [CookieExtensions.AddCookie](xref:Microsoft.Extensions.DependencyInjection.CookieExtensions.AddCookie*) | [CookieAuthenticationOptions.Cookie](xref:Microsoft.AspNetCore.Builder.CookieAuthenticationOptions.CookieName) | `Lax` |
+| <xref:Microsoft.Extensions.DependencyInjection.TwitterExtensions.AddTwitter*> | [TwitterOptions.StateCookie ](xref:Microsoft.AspNetCore.Authentication.Twitter.TwitterOptions.StateCookie) | `Lax`  |
+| <xref:Microsoft.AspNetCore.Authentication.RemoteAuthenticationHandler`1> | [RemoteAuthenticationOptions.CorrelationCookie](xref:Microsoft.AspNetCore.Authentication.RemoteAuthenticationOptions.CorrelationCookie)  | `None` |
+| <xref:Microsoft.Extensions.DependencyInjection.OpenIdConnectExtensions.AddOpenIdConnect*> | [OpenIdConnectOptions.NonceCookie](xref:Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectOptions.NonceCookie)| `None` |
+| <xref:Microsoft.AspNetCore.Http.CookieOptions> | A default cookie with a path of '/' | `None` |
+
+
+## APIs impacted by the SameSite 2019 draft standard
+
+* [Http.SameSiteMode](xref:Microsoft.AspNetCore.Http.SameSiteMode)
+* [CookieOptions.SameSite](xref:Microsoft.AspNetCore.Http.CookieOptions.SameSite)
+* [CookieBuilder.SameSite](xref:Microsoft.AspNetCore.Http.CookieBuilder.SameSite)
+* [CookiePolicyOptions.MinimumSameSitePolicy](xref:Microsoft.AspNetCore.Builder.CookiePolicyOptions.MinimumSameSitePolicy)
+* <xref:Microsoft.Net.Http.Headers.SameSiteMode?displayProperty=fullName>
+* <xref:Microsoft.Net.Http.Headers.SetCookieHeaderValue.SameSite?displayProperty=fullName>
+
+::: moniker range="= aspnetcore-3.1"
+
+ASP.NET Core 3.1 and later provides the following SameSite support:
+
+* Redefines the behavior of `SameSiteMode.None` to emit `SameSite=None`
+* Adds a new value `SameSiteMode.Unspecified` to omit the SameSite attribute.
+* All cookies APIs default to `Unspecified`. Some components that use cookies set values more specific to their scenarios, for example
+  * The `OpenIdConnect` correlation cookie.
+  * [OpenIdConnectOptions.NonceCookie](xref:Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectOptions.NonceCookie)
+
+::: moniker-end
+
+::: moniker range=">= aspnetcore-3.0"
+
+ASP.NET Core 3.0 and later aligns SameSite defaults with the new draft standard. The following APIs have changed the default from `SameSiteMode.Lax ` to `SameSiteMode.None`:
```

Comment:
The 3.0.1 patch changed these again from None to `(-1)`. Do you want to call that out separately or squash them to say the default changed from Lax to (-1)?